### PR TITLE
nikolai.berestevich/fix/help-centre-titles-highlight 36523

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:
-  styled component doesn't rerender with new props during the first loading of the page. So i added new  hook and variable to fix it 
-   change "href" to "to" in props of the ExternalLink  (as it should be - fix of  the empty pages in the /help-centre/dmt5)
## Type of change

-   [ x ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/help-centre/account.js [hunk 1]
```diff
@@ -144,7 +167,16 @@ const DormantFee = () => (
     </ArticleWrapper>
 )
 
+DormantFee.propTypes = {
+    children: PropTypes.any,
+    not_first_loading: PropTypes.bool,
+};
+
 const AccountArticle = () => {
+    const [not_first_loading, setNotFirstLoading] = useState(false) //needs to fix bug with hightlight of the 1st loading
```
Comment (sean-binary) [inline src/pages/help-centre/account.js:176]: Same minor renaming suggestion as to the other PR 🙏

**Review by sean-binary — DISMISSED** [review]
LGTM

**Review by kevinw-binary — DISMISSED** [review]

**Comment by nikolaiberestevich-deriv** [comment]
just rename of variables

File: src/pages/help-centre/trading.js [hunk 2]
```diff
@@ -55,6 +61,11 @@ const CFD = () => (
         </Text>
     </ArticleWrapper>
 )
+CFD.propTypes = {
+    children: PropTypes.any,
+    is_mounted: PropTypes.bool
+};
+
```
Comment (sean-binary) [inline src/pages/help-centre/trading.js:68]: Is this still needed?

File: src/pages/help-centre/trading.js [hunk 3]
```diff
@@ -112,51 +127,68 @@ const WeekendContracts = () => (
         <Text>{localize('Synthetic indices are available for trading 24/7.')}</Text>
     </ArticleWrapper>
 )
+WeekendContracts.propTypes = {
+    children: PropTypes.any,
+    is_mounted: PropTypes.bool
+};
```
Comment (sean-binary) [inline src/pages/help-centre/trading.js:133]: Same here.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**